spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+       </emu-alg>
+
+       <emu-table id="table-plural-operands">
+         <emu-caption>Internal Slots of Plural Rules Operands</emu-caption>
+         <table class="real-table">
+           <thead>
+             <tr>
+               <th>Internal Slot</th>
+               <th>Type</th>
+               <th>Description</th>
+             </tr>
+           </thead>
+           <tr>
+             <td>[[Number]]</td>
+             <td>Number</td>
+             <td>Absolute value of the source number (integer and decimals)</td>
+           </tr>
+           <tr>
+             <td>[[IntegerDigits]]</td>
+             <td>Number</td>
+             <td>Number of digits of [[Number]].</td>
+           </tr>
+           <tr>
+             <td>[[NumberOfFractionDigits]]</td>
+             <td>Number</td>
+             <td>Number of visible fraction digits in [[Number]], <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>[[NumberOfFractionDigitsWithoutTrailing]]</td>
+             <td>Number</td>
+             <td>Number of visible fraction digits in [[Number]], <i>without</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>[[FractionDigits]]</td>
+             <td>Number</td>
+             <td>Number of visible fractional digits in [[Number]], <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>[[FractionDigitsWithoutTrailing]]</td>
+             <td>Number</td>
+             <td>Number of visible fractional digits in [[Number]], <i>without</i> trailing zeros. </td>
+           </tr>
+         </table>
+       </emu-table>
+
+    </emu-clause>
+
+    <emu-clause id="sec-pluralruleselection" aoid="PluralRuleSelection">
+
+      <h1>PluralRuleSelection ( _locale_, _type_, _n_, _operands_ )</h1>
+
+      <p>
+        When the PluralRuleSelection abstract operation is called with four arguments, it performs implementation dependent algorithm to map _n_ to the appropriate plural representation of the Operand Record _operands_ by selecting the rules denoted by _type_ for the corresponding _locale_, or the String value `"other"`.
+      </p>
+
+    </emu-clause>
+
+    <emu-clause id="sec-resolveplural" aoid="ResolvePlural">
+      <h1>ResolvePlural ( _pluralRules_, _n_ )</h1>
+      <p>
+        When the ResolvePlural abstract operation is called with arguments _pluralRules_ (which must be an object initialized as a PluralRules) and _n_ (which must be a Number value), it returns a String value representing the plural form of _n_ according to the effective locale and the options of _pluralRules_. The following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Assert: Type(_pluralRules_) is Object.
+        1. Assert: If _pluralRules_ does not have an [[InitializedPluralRules]] internal slot, throw a *TypeError* exception.
```

Comment:
This assertion doesn't make sense, it's not possible to throw an error here. Instead:
`1. Assert: _pluralRules_ has an [[InitializedPluralRules]] internal slot.`